<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1642</width>
    <height>803</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="adjust_path_tab">
          <attribute name="title">
           <string>Adjust path</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <item>
              <layout class="QGridLayout" name="parameter_setting">
               <item row="13" column="0">
                <widget class="QLabel" name="label_beta">
                 <property name="text">
                  <string>Risk (beta)</string>
                 </property>
                </widget>
               </item>
               <item row="11" column="0">
                <widget class="QLabel" name="label_alpha">
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="text">
                  <string>Energy consumption (alpha)</string>
                 </property>
                </widget>
               </item>
               <item row="12" column="0">
                <widget class="QSlider" name="alpha_slider">
                 <property name="maximum">
                  <number>20</number>
                 </property>
                 <property name="singleStep">
                  <number>1</number>
                 </property>
                 <property name="value">
                  <number>20</number>
                 </property>
                 <property name="sliderPosition">
                  <number>20</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="tickPosition">
                  <enum>QSlider::TicksBelow</enum>
                 </property>
                </widget>
               </item>
               <item row="1" column="0" colspan="2">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Left click on map sets new start.</string>
                 </property>
                </widget>
               </item>
               <item row="12" column="1">
                <widget class="QLabel" name="value_alpha">
                 <property name="text">
                  <string>1.0</string>
                 </property>
                </widget>
               </item>
               <item row="16" column="0">
                <widget class="QSlider" name="gamma_slider">
                 <property name="maximum">
                  <number>20</number>
                 </property>
                 <property name="singleStep">
                  <number>1</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="tickPosition">
                  <enum>QSlider::TicksBelow</enum>
                 </property>
                </widget>
               </item>
               <item row="7" column="0">
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <widget class="QSpinBox" name="goalx">
                   <property name="maximum">
                    <number>1000</number>
                   </property>
                   <property name="value">
                    <number>0</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>m; </string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="goaly">
                   <property name="maximum">
                    <number>1000</number>
                   </property>
                   <property name="value">
                    <number>0</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_10">
                   <property name="text">
                    <string>m</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Start:</string>
                 </property>
                </widget>
               </item>
               <item row="8" column="0">
                <widget class="QPushButton" name="load_coordinates">
                 <property name="text">
                  <string>Load coordinates</string>
                 </property>
                </widget>
               </item>
               <item row="14" column="1">
                <widget class="QLabel" name="value_beta">
                 <property name="text">
                  <string>0.0</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Goal:</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="0">
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QSpinBox" name="startx">
                   <property name="minimum">
                    <number>0</number>
                   </property>
                   <property name="maximum">
                    <number>1000</number>
                   </property>
                   <property name="value">
                    <number>0</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>m; </string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="starty">
                   <property name="maximum">
                    <number>1000</number>
                   </property>
                   <property name="value">
                    <number>0</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>m</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="2" column="0" colspan="2">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Or input x-y-coordinates here:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0" colspan="2">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Right click on map sets new goal.</string>
                 </property>
                </widget>
               </item>
               <item row="14" column="0">
                <widget class="QSlider" name="beta_slider">
                 <property name="maximum">
                  <number>20</number>
                 </property>
                 <property name="singleStep">
                  <number>1</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="tickPosition">
                  <enum>QSlider::TicksBelow</enum>
                 </property>
                </widget>
               </item>
               <item row="16" column="1">
                <widget class="QLabel" name="value_gamma">
                 <property name="text">
                  <string>0.0</string>
                 </property>
                </widget>
               </item>
               <item row="10" column="0" colspan="2">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Please choose weights for cost function:</string>
                 </property>
                </widget>
               </item>
               <item row="9" column="0">
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="15" column="0">
                <widget class="QLabel" name="label_gamma">
                 <property name="text">
                  <string>Scientific interest (gamma)</string>
                 </property>
                </widget>
               </item>
               <item row="17" column="0">
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="plan_button">
               <property name="text">
                <string>Calculate path</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="1">
                <widget class="QPushButton" name="save_path1">
                 <property name="text">
                  <string>Save as path 2</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QPushButton" name="save_path0">
                 <property name="text">
                  <string>Save as path 1</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QPushButton" name="save_path3">
                 <property name="text">
                  <string>Save as path 4</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QPushButton" name="save_path2">
                 <property name="text">
                  <string>Save as path 3</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="pathshows_label">
               <property name="text">
                <string> </string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QWidget" name="map_showing" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QWidget" name="mapwidget_adjust" native="true">
                <property name="minimumSize">
                 <size>
                  <width>1300</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>1300</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QPushButton" name="map_down">
                  <property name="text">
                   <string>&lt;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="map_up">
                  <property name="text">
                   <string>&gt;</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="compare_paths_tab">
          <attribute name="title">
           <string>Compare several paths</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QTabWidget" name="compare_paths_tab_2">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="tab_pathoverview">
              <attribute name="title">
               <string>Overview</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_13">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_12">
                 <item>
                  <widget class="QWidget" name="img_paths" native="true">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>100</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <item>
                    <widget class="QPushButton" name="execute0">
                     <property name="text">
                      <string>Export path 1</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="execute1">
                     <property name="text">
                      <string>Export path 2</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="execute2">
                     <property name="text">
                      <string>Export path 3</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="execute3">
                     <property name="text">
                      <string>Export path 4</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QLabel" name="pathsaved_label">
                   <property name="text">
                    <string> </string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QTableWidget" name="results_table"/>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>216</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Choose Map Config File: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="openmapconfigbutton">
           <property name="text">
            <string>Open...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="mapload_label">
           <property name="text">
            <string>Map Config File: None</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Choose Robot Config File: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="openrobotconfigbutton">
           <property name="text">
            <string>Open...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="robotload_label">
           <property name="text">
            <string>Robot Config File: None</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>217</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="loadconfigbutton">
         <property name="text">
          <string>Load Configs</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>216</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
